src/client/Client.h

```diff
@@ -534,6 +534,8 @@ class Client : public Dispatcher, public md_config_obs_t {
    */
   void _handle_full_flag(int64_t pool);
 
+  void close_sessions();
```

Comment:
I can, but there are a tonne of functions in Client that don't follow that convention, so you might want to document it and update the code if you want to enforce it.